solidity/random-beacon/contracts/test/TestToken.sol

```diff
@@ -0,0 +1,10 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.5;
+
+import "@thesis/solidity-contracts/contracts/token/ERC20WithPermit.sol";
```

Comment:
Does it have to have an external dependency? Couldn't we have just an empty stub contract with no dependencies?

Reply:
Sure, we can.